sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyOptions.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents optional parameters for cryptographic operations.
+ */
+@Fluent
+public class CryptographyOptions {
+    /**
+     * Initialization vector for symmetric algorithms.
+     */
+    @JsonProperty(value = "iv")
+    private final byte[] initializationVector;
```

Comment:
As previously mentioned, it would be more intuitive to have specialized `EncryptOptions` and `DecryptOptions` that guides the user toward providing the right options. This requires much more knowledge about cryptographic methods.